pkg/orchestrator/model/types.go

```diff
@@ -20,6 +21,9 @@ var CheckName = "orchestrator"
 // ExtraLogContext is used to add check name into log context
 var ExtraLogContext = []interface{}{"check", CheckName}
 
+// NoExpiration is used for functions that take an expiration time
+var NoExpiration = cache.NoExpiration
```

Comment:
If possible we should avoid duplicating variables, it's already defined in `pkg/orchestrator/cache.go`